google-cloud-logging/lib/google/cloud/logging/metric/list.rb

```diff
@@ -72,7 +72,7 @@ def next
             return nil unless next?
             ensure_service!
             grpc = @service.list_metrics token: token, max: @max
-            self.class.from_grpc grpc, @service
+            self.class.from_grpc grpc, @service, @max
```

Comment:
Want to change the `max` param to keyword arg to match `Entry::List` and `Log::List`?

Reply:
Possibly, but I don't want to make a change like that in this PR.